src/Constants.js

```diff
@@ -144,19 +143,23 @@ export const BOSSES = [
     "Twinrova",
 ];
 
+export const LINKS_POCKET = [
```

Comment:
"Links Pocket" is a special location for the medallion/stone that you start with, so it makes better sense to keep it under Bosses. Starting items go in a separate part of the JSON.

Reply:
You're right, that would make a lot more sense. Will change that right now